src/activity_actor.cpp

```diff
@@ -2384,6 +2384,8 @@ void insert_item_activity_actor::finish( player_activity &act, Character &who )
                 success = holster->put_in( it, item_pocket::pocket_type::CONTAINER,
                                            /*unseal_pockets=*/true ).success();
                 if( success ) {
+                    who.add_msg_if_player( string_format( _( "You insert your %s into the %s." ),
```

Comment:
```suggestion
                    who.add_msg_if_player( string_format( _( "You insert your %1$s into the %2$s." ),
```
Positional arguments, so translators can rearrange as needed.

Reply:
ah thank you i was trying to find out how to do that